src/docs/asciidoc/wan/defining_wan_replication copy.adoc

```diff
@@ -1,64 +1,37 @@
 === Defining WAN Replication
```

Comment:
In general, I prefer a word "Configuring", but it might be a matter of taste. WDYT?

Reply:
There is a "Tuning WAN Replication" section. Configuring and tuning kind of point to the same concept. I feel right about "Defining"; we can brainstorm this further.